pkg/embed/manifests/pre-restore/vault-secrets-operator.jsonnet

```
@@ -25,6 +25,17 @@ local manifests = {
       vault: {
         address: vault_addr,
       },
+      priorityClassName: 'system-cluster-critical',
+      resources: {
+        limits: {
+          cpu: '100m',
+          memory: '128Mi',
+        },
```

```suggestion
           limits: self.requests,
```

nit: alias to requests?